modules/extensions-spring-support/src/main/java/org/mule/module/extension/internal/capability/xml/schema/builder/SchemaBuilder.java

```diff
@@ -612,14 +612,17 @@ private LocalComplexType generateMapComplexType(String name, final String descri
             public void onPojo()
             {
                 entryComplexType.getAttributeOrAttributeGroup().add(createAttribute(ATTRIBUTE_NAME_VALUE, valueType, false, SUPPORTED));
```

Comment:
if the child element is not generated, then the attribute should be required:
_createAttribute(ATTRIBUTE_NAME_VALUE, valueType, **true**, SUPPORTED)_